crates/ruff/src/rules/flake8_use_pathlib/violations.rs

```diff
@@ -9,7 +9,7 @@ define_violation!(
 impl Violation for PathlibAbspath {
     #[derive_message_formats]
     fn message(&self) -> String {
-        format!("`os.path.abspath` should be replaced by `.resolve()`")
+        format!("`os.path.abspath()` should be replaced by `Path.resolve()`")
```

Comment:
Ended up tweaking these a bit to match the terminology from this table: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module